common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -2772,4 +2773,13 @@ object CelebornConf extends Logging {
       .doc("The time before a cache item is removed.")
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("15s")
+
+  val MEMORY_PER_RESULT_PARTITION: ConfigEntry[String] =
+    buildConf("celeborn.network.memory.per.result.partition")
```

Comment:
use client prefix for this configuration

Reply:
modified to celeborn.client.network